jinahub/indexers/storage/PostgreSQLStorage/postgres_indexer.py

```diff
@@ -43,6 +43,7 @@ def __init__(
         max_connections=5,
         default_traversal_paths: List[str] = ['r'],
         default_return_embeddings: bool = True,
+        dry_run: bool = False,
```

Comment:
docstring is missing for `dry_run`
```suggestion
        dry_run: bool = False,
```